test/unit/uk/gov/hmrc/bindingtariffclassification/repository/CaseRepositorySpec.scala

```diff
@@ -141,8 +141,7 @@ class CaseRepositorySpec extends BaseMongoIndexSpec
     }
   }
 
-  // TODO: test all possible combinations
-  // TODO: the test scenarios titles need to be written and grouped properly
+  // TODO: we should mock the search mapper and avoid testing all possible scenarios - this is already covered in the integration tests
```

Comment:
do you agree on this comment?

Reply:
As mentioned above I think it’s important we test with the Mapper as there’s quite important Mongo logic in that